src/components/ContactsPage.js

```diff
@@ -14,23 +15,43 @@ class ContactsPage extends React.Component {
     this.setState({ contacts: [...data] });
   }
 
-  get getAppropriateJSX() {
+  changeIsChecked(id) {
+    const { contacts } = this.state;
+    const contact = contacts.find(item => item.id === id);
+    contact.isChecked = !contact.isChecked;
+    this.setState({ contacts });
+  }
+
+  updateLikes(id, step) {
+    const { contacts } = this.state;
+    contacts.find(item => item.id === id).likes += step;
+    this.setState({ contacts });
+  }
+
+  renderContactsPage() {
     const { contacts } = this.state;
     if (!contacts.length) return <InLineSpinner />;
     return (
       <div className="ui two column stackable center aligned grid">
         <div className="column">
-          <List array={contacts} text="Contacts" />
+          <ContactsList
+            contactsList={contacts}
+            title={TITLE.CONTACTS}
+            contactMethods={{
+              updateLikes: (id, step) => this.updateLikes(id, step),
+              changeIsChecked: id => this.changeIsChecked(id)
+            }}
+          />
         </div>
         <div className="column">
-          <List text="Favourites" />
+          <ContactsList title={TITLE.FAVOURITES} />
         </div>
       </div>
     );
   }
 
   render() {
-    return this.getAppropriateJSX;
+    return this.renderContactsPage();
```

Comment:
You can move all the code from renderContactsPage in the render() method. It's not worth it to have the render function only calling one function to render. It's ok to have seperate functions to render things if you have multiple stuff to render in the render() method. 

A suggestion would be to only make a seperate function for renderContactList:

```
renderContactList () => {
  <ContactsList
	            contactsList={contacts}
	            title={TITLE.CONTACTS}
	            contactMethods={{
	              updateLikes: (id, step) => this.updateLikes(id, step),
	              changeIsChecked: id => this.changeIsChecked(id)
	            }}
	          />
}
```